src/conf.py

```diff
@@ -67,6 +68,8 @@
 
 html_static_path = ["_static"]
 
+notfound_urls_prefix = None
```

Comment:
When building locally, I get the warning
```
The config value `notfound_urls_prefix' has type `NoneType', defaults to `str'.
make: *** [Makefile:26: html] Error 2
```
which is treated as an error which fails the build. 

Changing it to an empty string made the build succeed, but I do not know if it will have the same effect.